src/lib/operators/jit_operator/specialization/jit_repository.cpp

```diff
@@ -12,12 +12,6 @@
 
 namespace opossum {
 
-// Singleton
-JitRepository& JitRepository::get() {
-  static JitRepository instance;
```

Comment:
I added something to our lint script, is that what you had in mind? There is still another manually implemented singleton (`CurrentScheduler`) which does not get detected by the script and which also is not, as the other ones, trivially removed.

Reply:
What is the problem with the scheduler? Wouldn't this cause issues with plugins as well?

Reply:
I haven't really looked into that yet. The `CurrentScheduler` holds a private member `static std::shared_ptr<AbstractScheduler> _instance;` and hence `get()` returns a shared pointer instead of a reference.